tox.ini

```diff
@@ -120,21 +120,13 @@ commands = {[testenv:test-upstream-requirements-base]commands}
 # Linters
 [testenv:flake8]
 basepython = python3
-deps =
-    flake8
-    flake8-docstrings>=1.5.0
-    # https://github.com/JBKahn/flake8-print/pull/30
-    flake8-print>=3.1.0
-    flake8-bugbear
+deps = -rlinter-requirements.txt
 commands =
     flake8 src/aws_encryption_sdk/ setup.py
 
 [testenv:flake8-tests]
 basepython = {[testenv:flake8]basepython}
-deps =
-    flake8
-    # https://github.com/JBKahn/flake8-print/pull/30
-    flake8-print>=3.1.0
+deps = -rlinter-requirements.txt
```

Comment:
This is where things are going south. Flake8 is pretty generous, compared to flake8 with all the deps in linter-requirements... I vote we expand the ignore fields below to include all the errors from https://github.com/aws/aws-encryption-sdk-python/runs/4541757457?check_suite_focus=true

Can we do something like `--ignore D**`. That would be awesome...

Reply:
Yes, we can: https://flake8.pycqa.org/en/3.1.1/user/options.html#cmdoption-flake8--ignore

So `--ignore D` would work. In our case, `--ignore F811,E203,W503,D`

Reply:
I am pretty sure D is for documents, and I am okay with having a lower bar for test's documentation.

Reply:
Good catch. I'm still curious why this run failed but this one didn't: https://github.com/aws/aws-encryption-sdk-python/runs/4541406129?check_suite_focus=true

They appear to be using the same flake8 versions, but maybe I'm missing a transitive dependency that's different.

Reply:
I see, linter-requirements includes flake8-docstrings while the original flake8-tests target did not. I agree that the right path forward is to just ignore all doc stuff for the tests.